website/source/docs/drivers/docker.html.md

```diff
@@ -26,7 +26,7 @@ The `docker` driver supports the following configuration in the job specificatio
 * `network_mode` - (Optional) The network mode to be used for the container.
    Valid options are `default`, `bridge`, `host` or `none`. If nothing is
    specified, the container will start in `bridge` mode. The `container`
-   network mode is not supported right now.
+   network mode is not supported right now, this case also defaults to `bridge`.
```

Comment:
I like this idea better. If we have consensus on this, I can go ahead and implement this tomorrow.